src/background/messaging/rpc-methods/sign-stacks-transaction.ts

```diff
@@ -166,15 +186,13 @@ export async function rpcSignStacksTransaction(
   void trackRpcRequestSuccess({ endpoint: message.method });
 
   const requestParams = [
-    ['txHex', message.params.txHex],
+    ['txHex', getStacksTransactionHexFromRequest(message.params)],
     ['requestId', message.id],
     ['request', createUnsecuredToken(request)],
```

Comment:
I noticed I don't think we use the `request` from here anywhere, are we trying to avoid using tokens with these requests? We have a lot of default params to deal with, so I'm making some helpers on my branch.

Reply:
Nvm, I see where this is used now in `StacksTransactionSigner`.